privacy-dashboard/privacy-dashboard-impl/build.gradle

```diff
@@ -42,13 +44,20 @@ dependencies {
     implementation project(':feature-toggles-api')
     implementation project(':privacy-protections-popup-api')
     implementation project(':broken-site-api')
+    api AndroidX.dataStore.preferences
 
     implementation Google.dagger
     implementation AndroidX.appCompat
     implementation AndroidX.lifecycle.runtime.ktx
     implementation AndroidX.lifecycle.viewModelKtx
     implementation Google.android.material
 
+    // Room
+    implementation AndroidX.room.runtime
+    implementation AndroidX.room.rxJava2
+    implementation AndroidX.room.ktx
+    ksp AndroidX.room.compiler
+
```

Comment:
Unless I missed something, you're not using Room database anywhere, so these dependencies are redundant.